plugins/tenant/common/lib/nokul/tenant/codification/unit.rb

```diff
@@ -4,21 +4,33 @@ module Nokul
   module Tenant
     module Codification
       module Unit
-        mattr_accessor :const, default: ActiveSupport::InheritableOptions.new(gross_length: 3)
+        mattr_accessor :const, default: ActiveSupport::InheritableOptions.new(gross_length: 4)
 
         module_function
 
-        def code_generator(starting:, ending:, pattern: nil, memory:)
+        def code_generator(starting:, ending:, only: nil, except: nil, memory:)
```

Comment:
Gözümden kaçan bir örnek yoksa, örneklerin tümünde `memory: nil` kullanılmış.
Öntanımlı değeri `nil` yapmamak için bir neden var mıydı?
Varsa gözden kaçırmış olabilirim.

Reply:
Nedeni şöyle.  Memory önceden üretilen isimleri tutan nesne.  Bu nesnenin default'ı yok (her ne kadar nil'i yönetse de), ilgili tenant için bir yerden gelecek (veritabanı).  Örneklerde nil dememin nedeni örneği kolayca denemek için.